### PR TITLE
[eme] Update persistent-license tests, incuding DRM

### PR DESCRIPTION
**Pull request by mwatson2**
I have created separate tests for re-using a persisted license and re-using and then destroying it, so there are updates to the ClearKey tests too.

The DRM tests are not fully tested, as our server appears not to support the record of license destruction generated by Widevine on ChomeOS. We are working on a work-around for this, but in the meantime the tests are ready for review.

### CODE REVIEW

File: encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Successful Playback, persistent-license session with Clear Key, mp4</title>
```
Comment (jdsmith3000) [inline encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html]: nit:  add "destroy the license"

File: encrypted-media/clearkey-mp4-playback-retrieve-destroy-persistent-license.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+
+    <title>Encrypted Media Extensions: Successful Playback, persistent-license session with Clear Key, mp4, event sequence</title>
```
Comment (jdsmith3000) [inline encrypted-media/clearkey-mp4-playback-retrieve-destroy-persistent-license.html]: nit: add ", retrieve license, playback and destroy the license"

File: encrypted-media/clearkey-mp4-playback-retrieve-persistent-license.html [hunk 3]
```diff
@@ -2,7 +2,6 @@
 <html>
   <head>
     <meta charset=utf-8>
-    <meta name="timeout" content="long">
     <title>Encrypted Media Extensions: Successful Playback, persistent-license session with Clear Key, mp4, event sequence</title>
```
Comment (jdsmith3000) [inline encrypted-media/clearkey-mp4-playback-retrieve-persistent-license.html]: nit: add ", retrieve the license, playback"

File: encrypted-media/resources/clearkey-retrieve-destroy-persistent-license.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Retrieve stored persistent-license</title>
```
Comment (jdsmith3000) [inline encrypted-media/resources/clearkey-retrieve-destroy-persistent-license.html]: nit: add "Clear Key," and "Destroy the license"

File: encrypted-media/resources/clearkey-retrieve-persistent-license.html [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset=utf-8>
     <meta name="timeout" content="long">
-    <title>Encrypted Media Extensions: Retrieve stored persistent-license</title>
+    <title>Encrypted Media Extensions: Playback using retrieved stored persistent-license</title>
```
Comment (jdsmith3000) [inline encrypted-media/resources/clearkey-retrieve-persistent-license.html]: nit: add "Clear Key,"

File: encrypted-media/resources/drm-retrieve-destroy-persistent-license.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Retrieve stored persistent-license</title>
```
Comment (jdsmith3000) [inline encrypted-media/resources/drm-retrieve-destroy-persistent-license.html]: nit: add "DRM," and "destroy the license"

File: encrypted-media/resources/drm-retrieve-persistent-license.html [hunk 7]
```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Retrieve stored persistent-license</title>
```
Comment (jdsmith3000) [inline encrypted-media/resources/drm-retrieve-persistent-license.html]: nit: add "DRM,"

File: encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```
Comment (jdsmith3000) [inline encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html]: A few tests don't have this.  It that intentional?

`<meta name="timeout" content="long"> `

**Comment by jdsmith3000** [comment]
All small items noted.  Other than them, LGTM.

File: encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html (hunk 8 above)
Comment (mwatson2) [inline encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html]: Yes, none of them should need it.

Originally, I had many tests in one file and this took a long time, so we needed the long timeout. I'm trying to make them faster now and the long timeout should not be needed.

**Pull request merged**